### PR TITLE
feat: microsoft stream

### PR DESCRIPTION
**Pull request by onhate**

### CODE REVIEW

**Comment by onhate** [comment]
@radiovisual what do you think?

File: tests/index.test.js [hunk 1]
```diff
@@ -344,3 +344,36 @@ test('google link returns null as id and service if missing url parameter', (t)
...
+ *
+ */
+
+test('gets microsoft stream from url', (t) => {
+  t.is(fn('https://web.microsoftstream.com/video/73043e0c-cba8-482a-8a8e-0b72bc580ff0').id, '73043e0c-cba8-482a-8a8e-0b72bc580ff0');
+  t.is(fn('https://web.microsoftstream.com/video/73043e0c-cba8-482a-8a8e-0b72bc580ff0?list=trending').id, '73043e0c-cba8-482a-8a8e-0b72bc580ff0');
+  t.is(fn('https://web.microsoftstream.com/embed/video/73043e0c-cba8-482a-8a8e-0b72bc580ff0').id, '73043e0c-cba8-482a-8a8e-0b72bc580ff0');
+
+  t.is(fn('https://web.microsoftstream.com/video/73043e0c-cba8-482a-8a8e-0b72bc580ff0').service, 'microsoftstream');
+  t.is(fn('https://web.microsoftstream.com/embed/video/73043e0c-cba8-482a-8a8e-0b72bc580ff0').service, 'microsoftstream');
```
Comment (radiovisual) [inline tests/index.test.js:365]: Maybe these tests become easier to read/troubleshoot if the ids are smaller. Not super important though.

**Review by radiovisual — CHANGES_REQUESTED** [review]
Looks great from my point of view. Thanks for contributing @onhate. Could you please update the README with the patterns, as well as the package description to include a mention of Microsoft Streams?

Thanks again!

File: tests/index.test.js (hunk 1 above)
Comment (onhate) [reply tests/index.test.js:365]: sure thing, I just wanted to make sure it was closer to the real scenarios for the regex extraction

File: readme.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # get-video-id [![Build Status](https://travis-ci.org/radiovisual/get-video-id.svg?branch=master)](https://travis-ci.org/radiovisual/get-video-id) [![Coverage Status](https://coveralls.io/repos/github/radiovisual/get-video-id/badge.svg?branch=master)](https://coveralls.io/github/radiovisual/get-video-id?branch=master)
 
-> Get the YouTube, Vimeo, Vine or VideoPress video id from a url or embed string.
+> Get the YouTube, Vimeo, Vine, VideoPress or Microsoft Stream video id from an url or embed string.
```
Comment (radiovisual) [inline readme.md:3]: `a` is still the correct choice here. 
```suggestion
> Get the YouTube, Vimeo, Vine, VideoPress or Microsoft Stream video id from a url or embed string.
```

File: readme.md [hunk 3]
```diff
@@ -199,6 +199,19 @@ https://videopress.com/embed/*
...
+
+**Microsoft Stream urls**
+```
+https://web.microsoftstream.com/video/*
+https://web.microsoftstream.com/embed/video/*
+```
+
+**Microsoft Stream iframes**
+```
+<iframe src="https://web.microsoftstream.com/embed/video/73043e0c-cba8-482a-8a8e-0b72bc580ff4?autoplay=false&amp;showinfo=true" width="640" height="360"></iframe>
```
Comment (radiovisual) [inline readme.md:212]: To clean this up a bit, and follow the convention set by the other iframe examples, you could use something like this:
```suggestion
<iframe src="https://web.microsoftstream.com/embed/video/*?&" width="640" height="360"></iframe>
```

**Review by radiovisual — CHANGES_REQUESTED** [review]
Looks awesome. Just a few super small tweaks and it is ready to go. Thanks!

**Comment by onhate** [comment]
@radiovisual nice! ready to go :)

**Review by radiovisual — APPROVED** [review]
Great! Thanks so much! Let's ship this! 🚢 🚀

**Comment by radiovisual** [comment]
Microsoft Stream support has been published and now available in `v3.2.0` 🥳

**Comment by onhate** [comment]
great 🥳

**Pull request merged**